test/parallel/test-buffer-alloc.js

```diff
@@ -1060,6 +1060,8 @@ assert.throws(function() {
   Buffer.allocUnsafe(0xFFFFFFFFF);
 }, RangeError);
 
+assert(Buffer.alloc.toString().length < 600, 'Buffer.alloc is not inlineable');
```

Comment:
> I'm wondering though since it's not really a test per se that we might just put all of these kinds of checks in a separate file where we check the lengths of various commonly used functions?

I think this is a better solution but I'm not sure how to build the list of functions that should be checked.

WRT this particular instance I'm neutral, it doesn't harm to keep it (regression test) and it doesn't harm to remove it as long as we know the length is < 600 chars and we know we are going to have a dedicated file for these checks.